python/mlcroissant/mlcroissant/_src/structure_graph/nodes/file_object.py

```diff
@@ -68,16 +85,20 @@ def from_jsonld(
         )
         content_url = file_object.get(constants.SCHEMA_ORG_CONTENT_URL)
         name = file_object.get(constants.SCHEMA_ORG_NAME, "")
+
+        contained_in = box_singleton_list(
+            file_object.get(constants.SCHEMA_ORG_CONTAINED_IN)
+        )
+        if contained_in is not None and not ctx.is_v0():
```

Comment:
```
if ctx.is_v0():
  contained_in =
else:
  contained_in =
```

Reply:
(For readability)

Reply:
You mean to avoid the `if ... and ...` or to avoid the `not ctx._is_v0()` ?